dockerfiles/web/Dockerfile

```
@@ -20,7 +20,7 @@ ENV PYTHONUNBUFFERED 1
 
 RUN mkdir -p /opt/poetry /app /static /opt/static /dbox/Dropbox/media
 RUN python -m pip install -U pip
-RUN python -m pip install -U poetry
+RUN python -m pip install -U poetry==0.12.7
```

There is a more recent version:
```suggestion
RUN python -m pip install -U poetry==0.12.17
```